packages/components/src/Input/Input.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import { PropTypes } from 'prop-types'
+import styled from 'styled-components'
+
+import colors from '../colors'
+
+const { errorRed, black } = colors
+
+const Wrapper = styled.div`
+  margin-bottom: 20px;
+  display: flex;
+  font-family: 'Open Sans', sans-serif;
+  min-height: 60px;
+`
+
+const InputWrapper = styled.div`
+  width: ${({ type }) => (type === 'date' ? '15%' : '80%')};
+  display: flex;
+  flex-direction: column;
+`
+
+const Label = styled.label`
+  width: 20%;
+  max-width: 20%;
+  text-transform: uppercase;
+  overflow-wrap: break-word;
+  font-weight: 600;
+`
+
+const StyledInput = styled.input`
+  border: 1px solid ${black};
+  height: 30px;
+  padding: 5px;
+`
+
+const Currency = styled.span`
+  margin-left: 5px;
+`
+
+const ErrorMessage = styled.p`
+  color: ${errorRed};
+  display: block;
+  height: 18px;
+  margin: 2px 0;
+  font-size: 12px;
+`
+
+const MonetaryInput = ({ id, ...rest }) => (
+  <div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center' }}>
+    <StyledInput id={id} type="number" {...rest} />
+    <Currency>GBP</Currency>
+  </div>
+)
+
+const Input = ({ label, id, monetary, type, error, ...rest }) => (
```

Comment:
Replace `monetary` by `currency` which take a string value

Reply:
I think monetary should stay because that is defining a different input field in the UI.
This is the minimum requirements for this component, in the future, there should be an array of currencies and a dropdown to select one of them. for now, the currency is hardcoded as GBP as it was in the net-zero app.